cmd/operator/kodata/knative-eventing/0.17.0/2-eventing-core.yaml

```diff
@@ -708,9 +708,9 @@ spec:
           value: "8443"
           # SINK_BINDING_SELECTION_MODE specifies the NamespaceSelector and ObjectSelector
           # for the sinkbinding webhook.
-          # If `inclusion` is selected, namespaces/objects labled as `bindings.knative.dev/include:true`
+          # If `inclusion` is selected, namespaces/objects labelled as `bindings.knative.dev/include:true`
           # will be considered by the sinkbinding webhook;
-          # If `exclusion` is selected, namespaces/objects labled as `bindings.knative.dev/exclude:true`
+          # If `exclusion` is selected, namespaces/objects labelled as `bindings.knative.dev/exclude:true`
```

Comment:
I think that if these lived in `third_party` and were symlinked into kodata that we'd avoid these PRs.

cc @evankanderson @houshengbo 

/hold

Reply:
/hold

I thought that I'd avoided this with the exclusion of `linguist-generated` files: https://github.com/knative/operator/blob/master/.github/workflows/knative-style.yaml#L147

Let me see if I can figure out what's going on.

Reply:
Wait, is this being run by `prow`, instead of by GitHub actions?